### PR TITLE
93110 cpp contact query

### PR DESCRIPTION
**Pull request by numbap**
### Description

List of proposed changes:

- Contact page for CPP
- Unit tests for CPP contact page
- Unit tests for EI contact page
- GraphQL query for CPP contact page

### CODE REVIEW

File: components/contact/ContactProvinceRow.js [hunk 1]
```diff
@@ -1,29 +1,40 @@
...
-          </div>
-          <div className="col-span-1">
-            <p>
-              <b>{items[1].content}</b>
-            </p>
-            <Markdown>{`${items[1].recipient}\n\n${items[1].program}\n\n${items[1].poBox}\n\n${items[1].city} ${items[1].province} ${items[1].country}`}</Markdown>
-          </div>
+          {items.map((x, i) => (
+            <div className="col-span-1 py-3" key={i}>
+              <p>
```
Comment (shewood) [inline components/contact/ContactProvinceRow.js:19]: You could just do this for all of these bold text items   <p classname="font-bold">
               {x && x.content}
              </p>

Comment (numbap) [reply components/contact/ContactProvinceRow.js]: Updated.

File: components/contact/ContactSection.js [hunk 2]
```diff
@@ -15,8 +21,7 @@ export default function ContactSection({ title, intro, id, details }) {
           ContactSectionRow({ ...x, index, detail: x.items[0].content })
         )}
       </div>
-      <p></p>
-      <div className="grid grid-cols-4 gap-4 border-t-2 pb-6" />
+      <div className="grid grid-cols-4 gap-4 mt-4 border-t-2 pb-6" />
```
Comment (shewood) [inline components/contact/ContactSection.js:24]: What is the grid for here? Is there content supposed to be inserted at the bottom?

**Comment by shewood** [comment]
The Province names seem to be missing in French

File: components/contact/ContactSection.js (hunk 2 above)
Comment (numbap) [reply components/contact/ContactSection.js]: A grey line is supposed to be inserted here. I've removed the grid from the styles.

**Review by shewood — APPROVED** [review]
Looks good. 
There are still some issues with mobile view and accessibility but those can be resolved in another PR.

**Pull request merged**